src/Dobble/Deck.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Marmelab\Dobble;
+
+/**
+ * @todo validate & generate functions
+ */
+class Deck implements \Countable
+{
+    private $cards = [];
+
+    public function __construct($cards = [])
+    {
+        if (!is_array($cards) && !$cards instanceof \Traversable) {
```

Comment:
I'm ok for this use case, but what is the solution if we want to pass all types of iterable ?
An array is sadly not a [Traversable](http://php.net/Traversable) but all other objects which can pass into a foreach are.